Sources/SwiftFusion/Inference/ProbablisticTrackingFactor.swift

```diff
@@ -0,0 +1,70 @@
+import PenguinParallel
+import PenguinStructures
+import TensorFlow
+
+extension AppearanceModelEncoder {
```

Comment:
doc. What is this?

Reply:
removed, not relevant for the new factor